RFunction.R

```diff
@@ -398,3 +399,66 @@ rFunction = function(data,
   result <- data
   return(result)
 }
+
+
+
+
+
+
+# //////////////////////////////////////////////////////////////////////////////
+# using speed (i.e. a combination of time and location) as a proxy to identify
+# outliers in movement data (due to e.g. GPS glitches). All location events
+# associated with speeds above the user-defined threshold (given by some
+# sensible estimate of nonsensical speeds for the tagged species) asre removed
+# from the data.
+
+drop_outliers <- function(data, tresh_kmph){
```

Comment:
for consistency you might want to change `tresh` to `thresh` (just the missing h)

Reply:
yep: typo-alert! 

Fixed in [7a066dd](https://github.com/dmpstats/Standardise_Formats_and_Calculate_Basic_Statistics/pull/6/commits/7a066dd340682a34f7420f2e393a593b81f428ef)